### PR TITLE
fix: parse account name correctly for subledgers

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: src/lib/plugin.js [hunk 1]
```diff
@@ -331,7 +331,8 @@ class FiveBellsLedger extends EventEmitter2 {
   }
 
   * _send (transfer) {
-    const sourceAddress = parseAddress(transfer.account)
+    // Strip off the prefix and request id.
+    const sourceUsername = transfer.account.slice(this.prefix.length).split('.')[0]
```
Comment (emschwartz) [inline src/lib/plugin.js:337]: This behavior should be documented somewhere

Comment (emschwartz) [inline src/lib/plugin.js:337]: It would also be good to add a test for it

**Comment by emschwartz** [comment]
LGTM. @justmoon might want to review to make sure he agrees with this functionality

**Comment by justmoon** [comment]
Whoops, didn't see this one. #32

**Pull request closed without merging**